Tests/Tests/ABTesting/ExPlatTests.swift

```diff
@@ -103,6 +116,58 @@ class ExPlatTests: XCTestCase {
 
         XCTAssertEqual(ExPlat.shared?.experimentNames, ["foo", "bar"])
     }
+
+    // Tests ExPlat anonymous configuration using TracksService.
+    //
+    func testTracksServiceAnonymousConfiguration() {
+        // Given
+        let eventNamePrefix = "wooios"
+        let anonymousId = "123"
+        self.tracksService.platform = nil
+        self.tracksService.eventNamePrefix = eventNamePrefix
+
+        // When
+        self.tracksService.switchToAnonymousUser(withAnonymousID: anonymousId)
+
+        // Then
+#if os(iOS)
+        let exPlat = ExPlat.shared
+        XCTAssertNotNil(exPlat)
+        XCTAssertEqual(exPlat?.platform, eventNamePrefix)
+        XCTAssertEqual(exPlat?.oAuthToken, nil)
+        XCTAssertEqual(exPlat?.anonId, anonymousId)
```

Comment:
I think this code will change following @crazytonyli's suggestion [above](https://github.com/Automattic/Automattic-Tracks-iOS/pull/253/files#r1179889461), but I just wanted to ask if you considered using `XCTUnwrap` to get what I think would be clearer test failures if `exPlat` was nil.

```suggestion
        let exPlat = try XCTUnwrap(ExPlat.shared)
        XCTAssertEqual(exPlat.platform, eventNamePrefix)
        XCTAssertEqual(exPlat.oAuthToken, nil)
        XCTAssertEqual(exPlat.anonId, anonymousId)
```

Reply:
@mokagio Resolved in ec0af51dbd1c3b28876e995bfead11317f8815a2